datalad/support/annexrepo.py

```diff
@@ -499,3 +513,29 @@ def annex_whereis(self, files):
             item.get('file'):
                 [remote.get('description') for remote in item.get('whereis')]
                 for item in json_objects if item.get('success')}
+
+    @normalize_paths
+    def migrate_backend(self, files, backend=None):
+        """Changes the backend used for `file`.
+
+        The backend used for the key-value of `files`. Only files currently
+        present are migrated.
+        Note: There will be no notification if migrating fails due to the
+        absence of a file's content!
+
+        Parameters:
+        -----------
+        files: list
+            files to migrate.
+        backend: str
+            specify the backend to migrate to. If none is given, the
+            default backend of this instance will be used.
+        """
+
+        if self.is_direct_mode():
+            raise CommandNotAvailableError(
+                'git-annex migrate',
+                "Command 'migrate' is not available in direct mode.")
```

Comment:
Neither did I :)
But doesn't matter, the main reason we need for leads to migrating before the repo is cloned to a windows system anyway. And use MD5 as default when creating handles on windows, of course.